ppr-ui/src/resources/unitNotes.ts

```diff
@@ -28,8 +28,7 @@ export const NoticeOfTaxSaleDropDown: Array<UnitNoteDocTypes> = [
 
 // Dropdown items for Staff when Mhr has Residential Exemption note
 export const ResidentialExemptionStaffDropDown: Array<UnitNoteDocTypes> = [
-  UnitNoteDocTypes.PUBLIC_NOTE,
-  UnitNoteDocTypes.CONFIDENTIAL_NOTE
+  UnitNoteDocTypes.PUBLIC_NOTE
```

Comment:
Why was the second unit note type removed from the dropdown? Updated requirement? I didn't see it in figma).

Reply:
Based on figma, it said that "only show 'public note' in unit note dropdown menu. staff can still view filed unit notes, but only file a public note".

Reply:
OK sounds good, thanks